<inh f='llvm/build/include/c++/v1/stdexcept' l='76' c='std::logic_error'/>
<def f='llvm/build/include/c++/v1/future' l='498' ll='512'/>
<use f='llvm/build/include/c++/v1/future' l='510' c='_ZNSt3__112future_errorC1ERKS0_'/>
<use f='llvm/build/include/c++/v1/future' l='521' c='_ZNSt3__120__throw_future_errorENS_11future_errcE'/>
<use f='llvm/build/include/c++/v1/future' l='1342' c='_ZNSt3__17promiseD1Ev'/>
<use f='llvm/build/include/c++/v1/future' l='1485' c='_ZNSt3__17promiseIRT_ED1Ev'/>
<use f='llvm/libcxx/src/future.cpp' l='211' c='_ZNSt3__17promiseIvED1Ev'/>
<size>32</size>
<mbr r='std::__1::future_error::__ec_' o='128' t='std::__1::error_code'/>
<fun r='_ZNSt3__112future_errorC1ENS_10error_codeE'/>
<fun r='_ZNKSt3__112future_error4codeEv'/>
<fun r='_ZNSt3__112future_errorC1ERKS0_'/>
<fun r='_ZNSt3__112future_errorD1Ev'/>
<fun r='_ZNSt3__112future_errorC1ENS_10error_codeE'/>
<fun r='_ZNSt3__112future_errorD1Ev'/>
